crates/abq_cli/tests/cli.rs

```diff
@@ -2645,6 +2905,197 @@ fn report_while_run_in_progress_is_error() {
     term(queue_proc);
 }
 
+#[test]
+#[with_protocol_version]
+#[serial]
+fn test_explicit_run_id_against_ephemeral_queue() {
```

Comment:
I don't think we need this long of a test here. I think doing something like

```
        let CmdOutput {
            exit_status,
            stderr,
            stdout,
        } = Abq::new(format!("{name}_initial")).args(["--run-id", "test-run-id", "--", "false"]).run();
```
and checking the error output would be enough.